content/docs/studio/get-started.md

```diff
@@ -0,0 +1,63 @@
+# Get Started with DVC Studio
+
+Here, we will walk you through a tutorial to use DVC Studio for collaboration on
+your ML projects. You will need access to a Github, Gitlab or Bitbucket account
+which has access to the Git repositories you want to connect to from DVC Studio.
+DVC Studio creates views from repositories when you connect to them.
+
+## What is a view?
+
+A _view_ is an interactive representation of the experiments run in your ML
+project. DVC Studio automatically identifies datasets, metrics and
+hyperparameters in your ML experiments. You can also add custom files with the
+required data (more on this later). Using this data, DVC Studio creates a
+summary view. This includes tables of all the metrics across all your
+experiments. You can also generate plots and compare experiments here.
+
+## Access your first view
+
+In your browser, open <https://viewer.iterative.ai>. Sign in with your Github,
+Gitlab, or Bitbucket account.
+
+![](/img/studio/login_home.png) _When you first login, an example view is
+already created for you to explore, and you can add more views._
+
+When you first login, you will find that there already exists a view connecting
+to an example DVC project. Use this view to explore the features that DVC Studio
+has to offer.
+
+DVC Studio automatically identifies datasets, metrics and hyperparameters in
+your ML experiments. Each view on the dashboard displays the metrics. In the
+figure above, you can see that `avg_prec` and `roc_auc` metrics are displayed.
+
+## Components of a view
+
+You can dive deep into all the experiments committed to the repo. For this, open
+the view by clicking on the view name (in this case, `example-get-started`).
+
+A table will be generated as shown below. This includes metrics, hyperparameters
+and information about the datasets. All the data is flattened and neatly
+presented for you to evaluate and compare the experiments.
+
+![](/img/studio/main.png)
```

Comment:
It's pretty hard to read anything in that image. I also noticed it's used in the index too. It's probably OK in the index but here we would need a smaller example or to filter out many of the columns so it's readable to this proportion more or less:

![image](https://user-images.githubusercontent.com/1477535/117898537-c4af8400-b28a-11eb-961f-062c44aa82ce.png)

Probably no need to show **Plots** yet in this first figure.

Reply:
> For context: how it looks now (https://dvc-org-studio-docs-vntpxfoqdz.herokuapp.com/doc/studio/get-started#components-of-a-view):

![image](https://user-images.githubusercontent.com/1477535/117898604-f1fc3200-b28a-11eb-81b6-7dc500e3d8be.png)